Library/Formula/strider.rb

```diff
@@ -0,0 +1,93 @@
+class UniversalNpm < Requirement
+  fatal true
+  satisfy { which("npm") }
+  def message
+    "npm is required. If you have installed node with `--without-npm` option, reinstall with `--with-npm`."
+  end
+end
+
+class UniversalNode < Requirement
```

Comment:
Yeah. I don't love it being there either to be honest :grinning:. Feel free to leave it in for now, If the maintainers previously okayed the language it's probably still fine.